### ISSUE TITLE
Firefox automatically scrolls chat window back to top

### ISSUE THREAD

**Issue by jacksgt** [Nordaaker/convos#324]
Hi,

I'm having an issue with various Firefox version (Firefox ESR 45 on Debian Stretch, Firefox 51 and 52 on Windows 7), where the chat window automatically scrolls back to the top. It does however not happen with Chromium or Internet Explorer.
A recording of the problem can be found here: https://u.teknik.io/oloDE.mp4

I'm using convos 0.99_27

EDIT: This also happens when a new message arrives.

**Comment by jhthorsen** [comment]
This is now fixed in 0.99_28. The problem was that I tried to set an element attribute, which is in conflict with a method name:

```
-            $message.closest(".scroll-element").get(0).scrollTo = $message.get(0).offsetTop;
+            $message.closest(".scroll-element").get(0).nextTickScrollTop = $message.get(0).offsetTop;
```

**Comment by jacksgt** [comment]
Thanks for the super quick fix!
